templates/python/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

Comment:
Do tags always start with a `v` in Python Poetry repos? Why not allow all tags here?

Reply:
I couldn't answer tbh. Are you suggesting that we should allow all strings as in ["(.*)?"] ?

Reply:
@disrupted What do you think? You have more experience with Python projects?

Reply:
it is certainly not a convention for Python projects that it should use semantic versioning with the `v` prefix. I believe for our projects it is the case though.
If we're sure that we will only use tags for official releases, then we can remove the condition there.

```suggestion
    tags:
```

Reply:
Sounds good :+1: